debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -999,10 +1003,11 @@ public void shouldPropagateSourceColumnTypeScaleToSchemaParameter() throws Excep
     @Test
     @FixFor("DBZ-800")
     public void shouldReceiveHeartbeatAlsoWhenChangingNonWhitelistedTable() throws Exception {
-        // the low heartbeat interval should make sure that a heartbeat message is emitted after each change record
-        // received from Postgres
+        // the high heartbeat interval should make sure that a heartbeat message is emitted only
+        // after insert statement which allows to check that lsn is not flushed by itself
+        // but only when heartbeat message is produced
         startConnector(config -> config
-                .with(Heartbeat.HEARTBEAT_INTERVAL, "1")
+                .with(Heartbeat.HEARTBEAT_INTERVAL, "10000")
```

Comment:
@igabaydulin Does this mean that the test will always run for at least ten seconds?

Reply:
@jpechane Yes, it should have worked with 2-3 seconds interval too, but I decided to take more time to be sure that consumed heartbeat contains updated lsn. I have updated the test and reduced the heartbeat interval to 100 ms. The test should run for 3-4 seconds now.